netlify.toml

```diff
@@ -1,11 +0,0 @@
-[build]
-    base = "Doc/"
-    command = "make html"
-    publish = "build/html"
-    # Do not trigger netlify builds if docs were not changed.
-    # Changed files should be in sync with `.github/workflows/doc.yml`
-    ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../netlify.toml"
```

Comment:
You can keep something like this on RTD (cancel the build if there is no changes at `Doc/` directory, for example) by running a test command that exits with code `183`. Take a look at https://docs.readthedocs.io/en/latest/builds.html#cancelling-builds for more information.

We have something like this at Read the Docs itself in our config file: https://github.com/readthedocs/readthedocs.org/blob/main/.readthedocs.yml#L28

Reply:
Attempted in https://github.com/python/cpython/pull/104100.